skate-plugin/project-gen/src/jvmMain/kotlin/slack/tooling/projectgen/UiElement.kt

```diff
@@ -35,15 +35,19 @@ internal object DividerElement : UiElement {
 }
 
 @Immutable
-internal data class SectionElement(val title: String, val description: String) : UiElement {
+internal data class SectionElement(
+  val title: String,
+  val description: String,
+  var indentLevel: Int = 0,
+) : UiElement {
   override var isVisible: Boolean = true
 }
 
 internal class CheckboxElement(
   private val initialValue: Boolean,
   val name: String,
   val hint: String,
-  val indentLevel: Int = 0,
+  var indentLevel: Int = 0,
```

Comment:
If you want this to be mutable then we should make it a `State` variable like `isChecked` is below. Ideally we shouldn't need this though? Do we have checkboxes whose' indention is changing conditionally after creation?

Reply:
Updated back to `val` https://github.com/slackhq/slack-gradle-plugin/pull/799/commits/742142fe1bbe3cc3e2d96e29383f4c4b2b6ddab9